easybuild/easyconfigs/g/Geant4-data/Geant4-data-11.1.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'Tarball'
+name = 'Geant4-data'
+version = '11.1'  # version should somewhat match the Geant4 version it should be used in
+
+homepage = 'https://geant4.web.cern.ch/'
+description = """Datasets for Geant4."""
+
+toolchain = SYSTEM
+
+# Pick up the correct sets and versions from cmake/Modules/G4DatasetDefinitions.cmake
+# in the Geant source
+local_datasets = [
+    ('G4NDL', '4.7', 'G4NDL', 'G4NEUTRONHPDATA'),  # NDL
+    ('G4EMLOW', '8.2', 'G4EMLOW', 'G4LEDATA'),  # Low energy electromagnetics
+    ('PhotonEvaporation', '5.7', 'G4PhotonEvaporation', 'G4LEVELGAMMADATA'),  # Photon evaporation
+    ('RadioactiveDecay', '5.6', 'G4RadioactiveDecay', 'G4RADIOACTIVEDATA'),  # Radioisotopes
+    ('G4SAIDDATA', '2.0', 'G4SAIDDATA', 'G4SAIDXSDATA'),  # SAID
+    ('G4PARTICLEXS', '4.0', 'G4PARTICLEXS', 'G4PARTICLEXSDATA'),  # Particle XS - replaces Neutron XS
+    ('G4PII', '1.3', 'G4PII', 'G4PIIDATA'),  # PII
+    ('RealSurface', '2.2', 'G4RealSurface', 'G4REALSURFACEDATA'),  # Optical Surfaces
+    ('G4SAIDDATA', '2.0', 'G4SAIDDATA', 'G4SAIDXSDATA'),  # SAID
```

Comment:
`G4SAIDDATA` is listed twice, is that intentional?

Reply:
(name-of-dataset, version, name-part-of-topdir-of-unpacked-tarfile, envvarname)

So it's not listed twice.

Reply:
Oh... that I didn't notice... fixing.

Reply:
@smoors @jfgrimm better now?